tests/test_device.py

```diff
@@ -40,11 +45,27 @@
 
 [honeywell.hqs]
 shots = 99
-api_key = "{}"
+user_email = "{}"

```

Comment:
Perhaps change `SOME_API_KEY` to e.g. `DUMMY_EMAIL` or similar?

Reply:
Good idea! Updated it.